incubator/oauth-proxy/templates/configmap.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.config.configFile }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: {{ template "oauth-proxy.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ template "oauth-proxy.fullname" . }}
```

Comment:
`oauth2-proxy.fullname`

Reply:
I've taken care of this in PR: https://github.com/kubernetes/charts/pull/4744